portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/TranslationManager.path

```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+<title>TranslationManager</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">TranslationManager</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>SELECT_TRANSLATION_LANGUAGE_MENU</td>
+	<td>//div[h3='${key_title}']//button[contains(@title,'select-translation-language')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SELECT_TRANSLATION_LANGUAGE_AUI_TAG_ADMINMENU</td>
+	<td>//button[contains(@id,'aui_2d_2Menu')]//*[name()='svg'][contains(@class,'lexicon-icon-${key_currentLocale}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SELECT_TRANSLATION_LANGUAGE_MENU_ITEM_INDEX</td>
+	<td>//div[contains(@class,'dropdown-menu') and contains(@class,'show')]//ul[contains(@class,'list-unstyled')]//li[${key_index}]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>MANAGE_TRANSLATIONS</td>
+	<td>//button[contains(@class,'dropdown-item')]//div[contains(@class,'autofit-section')and text()='Manage Translations']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>INPUT_TEXT_FIELD</td>
+	<td>//div[contains(@class,'col') and h3='${key_title}']//input[contains(@class,'language-value field')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>INPUT_FORM_TEXT</td>
```

Comment:
For my understanding, this path is for the text of the default language that displays below the input box of AUI Tag > Admin field when the current language is non-default, right?

But the title is not very clear to me, I cannot understand which element it is on UI by just looking at the name, I have to take a look at all the steps in test. Could you update the name to a more specific one?

And also this is for AUI Tag only, I think it's better to include AUI in the name as well, just as this one `SELECT_TRANSLATION_LANGUAGE_AUI_TAG_ADMINMENU` because there is another admin input box of Liferay UI tag. Please update all other path names if needed such as `INPUT_TEXT_FIELD`. Or maybe you can group all the paths by React Component, AUI tag, and Liferay UI tag.